packages/react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -77,12 +77,6 @@ export const CheckboxBase: React.FunctionComponent<ICheckboxProps> = React.forwa
 
     const onRenderLabel = props.onRenderLabel || defaultLabelRenderer;
 
-    const ariaChecked: React.InputHTMLAttributes<HTMLInputElement>['aria-checked'] = isIndeterminate
```

Comment:
Just to confirm I'm understanding what you said in the description correctly: screen readers should automatically read the correct checked/unchecked/indeterminate state from just `checked` + `indeterminate`, right?

Reply:
Tried it out and at least with default settings, Narrator+Edge reads the indeterminate state, but NVDA+Firefox/Edge/Chrome doesn't (testing on Windows 11 beta, in case that matters)

Reply:
I tested on this example (both checkboxes are indeterminate, but only the second has `aria-checked="mixed"`): https://jsfiddle.net/apd715qu/show

All SRs I tested treat both the same. NVDA reads "half checked" for me for both in FF/Edge/Chrome. Yours might be different if you have a different version of NVDA or one of the browsers, but as long as both are treated the same, the `aria-checked` value isn't adding anything. It makes sense that both would be treated the same, since the API mappings are identical for both (though of course SRs might always do weird stuff with DOM scraping, though that's less common now).

TL;DR: yup, SRs should automatically read the right state from just `checked` + `indeterminate` :)